traffic_ops/traffic_ops_golang/systeminfo/system_info.go

```diff
@@ -38,10 +38,19 @@ func Get(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	defer inf.Close()
-	api.RespWriter(w, r, inf.Tx.Tx)(getSystemInfo(inf.Tx, inf.User.PrivLevel, time.Duration(inf.Config.DBQueryTimeoutSeconds)*time.Second))
+
+	canReadSecureValue := false
+	if (inf.Version.GreaterThanOrEqualTo(&api.Version{Major: 4}) && inf.Config.RoleBasedPermissions) || inf.Version.GreaterThanOrEqualTo(&api.Version{Major: 5}) {
+		if inf.User.Can("PARAMETER:SECURE-READ") {
```

Comment:
Make a `tc` constant for `PARAMETER:SECURE-READ`? I know we aren't doing this for other perms yet, but we have to start somewhere, and hard-coding strings like this makes it harder than it should be to find where perms are used.